apps/emqx_bridge_oracle/README.md

```diff
@@ -0,0 +1,12 @@
+# Oracle Database Bridge
+
+This application houses the Oracle Database bridge for EMQX Enterprise Edition.
+It provides the means to connect to Oracle Database and publish messages to it.
```

Comment:
it's identical to emqx_oracle/README.md
please update it to reflect the fact that this app implements the data bridge APIs